packages/components/src/notice/types.ts

```diff
@@ -115,6 +115,12 @@ export type NoticeProps = {
 	 * instead of a string.
 	 */
 	__unstableHTML?: boolean;
+	/**
+	 * How long the notice should be presented to users
+	 *
+	 * @default true
```

Comment:
Hi @badasswp, Thank you for the PR. 
I noticed a small issue in the JSDoc documentation for the `duration` prop. I tink this `@default` should be `10000` as it is of type `number`.

```suggestion
	/**
	 * How long the notice should be presented to users
	 *
	 * @default 10000
```